doc/rst/developer/bestPractices/CHPL_DEVELOPER.rst

```diff
@@ -39,7 +39,7 @@ possible.  In particular:
    We should fix this).
 
 5) Turns on ``--devel`` by default when invoking the ``chpl`` compiler.  This
-   in turn has the following effects:
+   in turn, has the following effects:
```

Comment:
This sentence should either read "This, in turn, has the following effects" or "This in turn has the following effects".  The way you have it now removes the subject ("This") from the verb ("has")